src/Tools/dotnet-dump/Dumper.cs

```diff
@@ -81,6 +81,9 @@ public int Collect(IConsole console, int processId, string output, bool diag, bo
                     case DumpTypeOption.Mini:
                         dumpTypeMessage = "dump";
                         break;
+                    case DumpTypeOption.Triage:
+                        dumpTypeMessage = "triage";
```

Comment:
Given the message template, this should probably be "triage dump"

Reply:
fixed.